### PR TITLE
FT-1778 - Fixing paper case share a case defaults

### PR DESCRIPTION
**Pull request by ocarleton**
### Change description ###

creating to make pods

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by ocarleton** [comment]
evidence
https://gateway-finrem-ccd-definitions-pr-257.service.core-compute-preview.internal/print/jurisdictions/DIVORCE/case-types/FinancialRemedyContested/cases/1602171993443992

policy setting
image

isadmin/paperapplication setting
image

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
       "ccdUrl": "http://ccd-data-store-api-demo.service.core-compute-demo.internal"
     },
     "aat": {
-      "cosUrl": "http://finrem-cos-aat.service.core-compute-aat.internal",
+      "cosUrl": "http://finrem-cos-pr-482.service.core-compute-preview.internal",
```
Comment (ocarleton) [inline package.json:22]: removing premerge

**Review by shaunganley — APPROVED** [review]

File: definitions/contested/json/CaseEvent/CaseEvent.json [hunk 2]
```diff
@@ -786,6 +786,7 @@
     "DisplayOrder": 122,
     "PostConditionState": "caseAdded",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/case-orchestration/contested/set-paper-case-defaults",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/case-orchestration/contested/set-paper-case-org-policy",
```
Comment (shaunganley) [inline definitions/contested/json/CaseEvent/CaseEvent.json:789]: any idea how this works now when it failed with the previous callback?

**Review by johnoneill46 — APPROVED** [review]

File: definitions/contested/json/CaseEvent/CaseEvent.json (hunk 2 above)
Comment (ocarleton) [reply definitions/contested/json/CaseEvent/CaseEvent.json:789]: It does set fields which are in event fields 
CallBackURLAboutToStartEvent can be used to save values
https://tools.hmcts.net/confluence/display/RCCD/Callbacks+%28Webhooks%29+documentation
"Can be used to pre-populate case data prior to submission."

I think the issue is it donesn't set items which are not in the caseEventToFields.
I did try adding paperapplication, isAdmin etc to the fields but hiding them from the screen but it didn't take. I'm thinking they might need to be displayed.
This was the quick workaround which sorts it, though in theory its doable the other way

**Pull request closed without merging**